### ISSUE TITLE
Latest version of lass not being used?

### ISSUE THREAD

**Issue by niftylettuce** [cabinjs/parse-app-info#1]
It looks like this was copied over from `parse-request` or `parse-err`.  The issue is that this package will only work server-side.

Could you simply do the following?

```sh
npm install -g lass@latest
```

```sh
cd ~/projects
mv parse-app-info parse-app-info-backup
lass parse-app-info
cd parse-app-info
```

Then copy the `~/projects/parse-app-info-backup/src/index.js` file to `~/projects/parse-app-info/index.js`.

Also you'll need to `npm install last-commit-log read-pkg-up` from the `~/projects/parse-app-info` folder.

Lastly you'll need to copy over your tests.

Once done push a new commit or just git push with `--force` flag?

**Comment by niftylettuce** [comment]
Edit: In the above comment I said "The issue is that this package will only work server-side."

What I meant was: This package is currently using browserify, however it should actually only be targeting the server.

What this means is: You can drop all the browserify/babelify/tinyify stuff from package.json.  But it's probably easiest to just follow my instructions above instead

**Comment by pke** [comment]
That's correct, I modelled the package after parse-err. But if browser support is not required, even better ;)
Updated the repo via force-push.

**Comment by niftylettuce** [comment]
@pke this is now perfect

can you grant me collaborator status and ownership on npm?

```sh
npm owner add niftylettuce
```

and add "niftylettuce" as collaborator to this repo

I'll then use `np` and `release` to publish a version

**Comment by niftylettuce** [comment]
actually @pke you will need to release first version before you can give me access on npm.  here's what you need to do:

```sh
npm install -g np release
```

then run

```sh
np --no-yarn && release
```

**Comment by niftylettuce** [comment]
@pke one more thing - can you make sure you have two-factor authentication enabled on both NPM and Github?

**Comment by niftylettuce** [comment]
thank you, looks good 👍! https://www.npmjs.com/package/parse-app-info

**Comment by pke** [comment]
2FA was enabled everywhere. I published the package and specifically set the package access requiring 2FA too. I added you as a maintainer to npm and as a collaborator to GH.

I did use normal `npm publish` as always, cause `np` asked me to increase the version number already for the first release.

**Comment by pke** [comment]
We could add a NPM badge to the readme. Or maybe even add this feature to lass in general.

**Comment by niftylettuce** [comment]
@pke done, I've released lass@1.0.9 with support for automatic npm downloads total inserted into README
